BSB_LAN/BSB_LAN.ino

```diff
@@ -9143,7 +9143,8 @@ boolean mqtt_connect()
       }
       else
       {
-        printlnToDebug(PSTR("Connect to MQTT broker"));
+        printlnToDebug(PSTR("Connect to MQTT broker, updating will topic"));
+        printFmtToDebug(PSTR("Will topic: %s"), MQTTWillTopic.c_str());
```

Comment:
please add \r \n for new line
printFmtToDebug(PSTR("Will topic: %s\r\n"), MQTTWillTopic.c_str());

Reply:
A damn, I saw that in my logs but forgot to fix it. you'll get an updated PR in a minute.